contributing/running-server-side-experiment.md

```diff
@@ -0,0 +1,128 @@
+
+This doc explains how to create a server side experiment.
+
+## Before you start
+
+Because real traffic can be automatically assigned to the experiment, plus the experiment slots are limited. Any experiment declarition and launch requires an issue tracker and at least one approval from @ampproject/wg-approvers or @ampprojeg/wg-ads depending on the type of experiment environment(#environment).
+
+## Run an Experiment
+
+### Declare an Experiment
+
+Declare an experiment flag that's used during transpiling process. An example looks like
+```
+// eslint-disable-next-line no-undef
+if (MY_EXPERIMENT) {
+  // new code path
+} else {
+  // old code path
+}
+```
+During transpiling, babel will replace the experiment flag with a boolean value and removed the unused code path.
+
+The experiment flag values are default to `false`. You can use the `--define_experiment_constant` flag to set the value to `true`.
+
+```
+gulp build --define_experiment_const MY_EXPERIMENT`
+```
+
+### Start an Experiment
+
+Once you receive approval to start a server side experiment. You can add your experiment to one of the three experiment slots under the `experiments-config.js` [file](https://github.com/ampproject/amphtml/blob/master/build-system/global-configs/experiments-config.json).
+
+Example
+```
+{
+  "experimentA": {
+    "name": "myExperimentName",
+    "environment": "AMP",
+    "issue": "Link to the experiment tracking issue",
+    "expiration_date_utc": "Date in YYYY-MM-DD format, in UTC",
+    "define_experiment_constant": "MY_EXPERIMENT"
+  },
+  "experimentB": {},
+  "experimentC": {}
+}
+```
+
+#### More on Experiment Config
+
+##### Environment
+
+There are two types of environment `AMP` and `INABOX`. Use `AMP` if your experiment applied to AMP Documents, use `INABOX` if you want to run the experiment within AMPHTML ads.
+
+##### Expiration Date UTC
+
+The experiment expiration date in YYYY-MM-DD format, in UTC. If an experiment is expired, it will fail the build process. This expiration date is only read during the build. As a result, the experiment will actually end on the following release date after the expiration.
+
+Please keep the expiration date 30 days within the start date unless otherwise instructed.
+
+### What's Happening Behind
+
+#### Building Experimental Binaries
+
+After a new experiment slot has been added, AMP's release framework will respect the config and build additional experimental binaries. E2E tests will be ran with these experiment binaries.
+
+Once the experimental bianries are shipped, once can access those with different RTV prefix.
+
+For Experiment with `environment = AMP`. The Beta version that has prefix (03) will be used as control build for all experiments.
```

Comment:
The control build (which is binary equivalent to stable) is used, so it's `02`

Reply:
We do want to use RC (03) as the control group because RC and the experimental build are promoted at the same time, while control build(02) is promoted a week after. 

I renamed the `control build` to `control group`, hope that clarify.